script/Ciel/Day 5/05_01_CIEL05_3.txt

```diff
@@ -177,6 +177,7 @@ Oh... then what about Ciel-senpai?"
 [sha:134866779fb40c366bffac5aa4dc2e092868a064]{
 -- Page 400, Offset 11942. Glued. Mods: @x.
 -- あの子も本職は代行者よ。目的は吸血鬼の殲滅。この任務が終われば次の任務先に向かう、さすらいの転校生ってワケ」
+// group pr check: あの子も本職は代行者よ she's also an executor or something along those lines
```

Comment:
I really don't have anything better than the current line and this was a pretty minor gripe.

Reply:
> She's actually an Executor too.

Is something along the lines of what I was thinking